client/app/index.html

```diff
@@ -14,6 +14,10 @@
     <link rel="stylesheet" href="scripts/bower_components/jcrop/css/jquery.Jcrop.css">
     <!-- endbuild -->
 
+    <!-- build:css styles/css/jqueryui.slider.css -->
+    <link rel="stylesheet" href="scripts/bower_components/jquery-ui/themes/base/slider.css">
```

Comment:
imo you can put those bower components css files into single build:css with name like `vendor.css`